src/applications/disability-benefits/686c-674/config/form.js

```diff
@@ -380,14 +378,16 @@ const formConfig = {
           schema: studentLastTerm.schema,
         },
         studentIncomeInformation: {
-          depends: () => false,
+          depends: formData =>
+            isChapterFieldRequired(formData, TASK_KEYS.report674),
           title: 'Information needed to add a student 18 to 23 years old',
           path: 'report-674-student-income-information',
           uiSchema: studentIncomeInformation.uiSchema,
           schema: studentIncomeInformation.schema,
         },
         studentNetworthInformation: {
-          depends: () => false,
```

Comment:
These two snippets need to be reverted as well. Income and net worth questions need to remain out until post-launch.